<!-- -->
<launch>

	<arg name="ars_path_follower_node_name" 	default="ars_path_follower_node" />
	<arg name="screen" 	default="screen" />


  <node name="$(arg ars_path_follower_node_name)" pkg="ars_path_follower" type="ars_path_follower_ros_node.py" output="$(arg screen)" >


  </node>


</launch>
